CHANGELOG.md

```diff
@@ -11,12 +11,14 @@
 - `TabBuilder` for the `BuiltTab`
 - Support for all implementations of `Into<WidgetText>` in tab titles
 - Style editor in the `hello` example
+- Added `Tree::find_tab`
```

Comment:
Include the addition of `TabViewer`, `DynamicTabViewer`, `DynamicTree`, and `text_editor` example.